components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/DefaultStepBasedSequenceHandlerTest.java

```diff
@@ -173,6 +171,7 @@ public void testGetServiceProviderMappedUserRoles(Map<String, String> spRoleMapp
 
     @DataProvider(name = "spRoleClaimUriProvider")
     private Object[][] getSpRoleClaimUriData() {
+        Util.mockIdentityUtil();
```

Comment:
It's good we have one line between method signature and logic. Please check other places too.

Reply:
Addressed with commit 1ab06b94b3607a2e2f94d5cfb23dbf0d5074d3cb